crates/cli-support/src/js/mod.rs

```diff
@@ -56,7 +56,7 @@ pub struct Context<'a> {
     /// wasm-bindgen emits.
     pub direct_imports: HashMap<&'a str, (&'a str, &'a str)>,
 
-    pub exported_classes: HashMap<String, ExportedClass>,
+    pub exported_classes: Option<HashMap<String, ExportedClass>>,
```

Comment:
Ah ok, this second commit is exactly the pattern that I was suggesting for the first one :-p